include/odp/api/spec/pool.h

```diff
@@ -176,6 +176,24 @@ int odp_pool_index(odp_pool_t pool);
  */
 int odp_pool_stats(odp_pool_t pool, odp_pool_stats_t *stats);
 
+/**
+ * Get selected statistics for pool handle
```

Comment:
Word "handle" can be dropped from this and odp_pool_stats()/odp_pool_stats_reset() summary lines.

This line could be simply "Get selected pool statistics"

Reply:
Changed in V2.